core/src/decoder/mod.rs

```diff
@@ -67,6 +67,9 @@ pub const OP_BATCH_1_GROUPS: [Felt; NUM_OP_BATCH_FLAGS] = [ZERO, ONE, ONE];
 /// Index of the op bits extra column in the decoder trace.
 pub const OP_BIT_EXTRA_COL_IDX: usize = OP_BATCH_FLAGS_RANGE.end;
 
+/// Index of the in_syscall column in the decoder trace.
```

Comment:
small question: every `core::ops::Range` is exclusive at end. If we are adding `1` to the exclusive range, it means we are in fact jumping two items in the set. Is that intended?

Reply:
This looks ok to me. `OP_BIT_EXTRA_COL_IDX` is set to the exclusive end value, following the `OP_BATCH_FLAGS_RANGE`, so nothing is jumped before `IN_SYSCALL_COL_IDX`. Have I misunderstood?